xbmc/GUIInfoManager.cpp

```diff
@@ -6989,6 +6989,8 @@ const infomap listitem_labels[]= {{ "thumb",            LISTITEM_THUMB },
                                   { "setid",            LISTITEM_SETID },
                                   { "videocodec",       LISTITEM_VIDEO_CODEC },
                                   { "videoresolution",  LISTITEM_VIDEO_RESOLUTION },
+                                  { "videowidth",       LISTITEM_VIDEO_WIDTH},
+                                  { "videoheight",      LISTITEM_VIDEO_HEIGHT},
```

Comment:
Please provide documentation for the new labels her: https://github.com/xbmc/xbmc/blob/36e4c0fd0cb92eac2bcd28e671ba26fb5aaafc59/xbmc/GUIInfoManager.cpp#L6874 ff.

Reply:
Where, please?

Reply:
Done